crypto/secp256k1eth/const.go

```diff
@@ -1,6 +1,9 @@
 package secp256k1eth
 
 const (
+	// Enabled indicates if this curve is enabled.
+	Enabled = false
```

Comment:
But this is part is built unconditionally, so you'll always have it set to false (even when you include flag `secp256k1eth`)... or am I missing something here?